ginivision/src/main/java/net/gini/android/vision/DocumentImportFileTypes.java

```diff
@@ -0,0 +1,6 @@
+package net.gini.android.vision;
+
+public enum  DocumentImportFileTypes {
```

Comment:
Maybe the `enum` name was a bit confusing for me, because when I read it I thought it will have `PDF`, `IMAGES` as cases. 
Since this `enum` is used for the file import toggle and as a suggestion, could it be better to name it as something like _DocumentImportSupportedFileTypes_ or _DocumentImportEnabledFileTypes_? 
And also add a case when any type is supported, instead of using `null`.